packages/x-date-pickers/src/TimePicker/TimePickerToolbar.tsx

```diff
@@ -79,7 +85,11 @@ const TimePickerToolbarHourMinuteLabel = styled('div', {
 const TimePickerToolbarAmPmSelection = styled('div', {
   name: 'MuiTimePickerToolbar',
   slot: 'AmPmSelection',
-  overridesResolver: (props, styles) => styles.ampmSelection,
+  overridesResolver: (props, styles) => [
+    { [`.${timePickerToolbarClasses.ampmLabel}`]: styles.ampmLabel },
```

Comment:
Did you forget the `&`?

Reply:
In this particular case - nope. Will test if it is needed on next line. 👌

Reply:
Did re-test, everything seems to be as needed to have correct overridden styles application. 🤞